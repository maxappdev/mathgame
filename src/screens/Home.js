import React from 'react'
import { 
    View, 
    Button } from 'react-native'
import styles from '../styles/HomeStyles'


const Home = (props) => {
    return (
        <>
            <View style={styles.container}>
            <Button title="Play" onPress={() => props.navigation.navigate('Game')} />
            </View>
        </>
    );
}
export default Home