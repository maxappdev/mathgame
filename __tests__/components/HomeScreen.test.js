import 'react-native';
import React from 'react';
import Home from '../../src/screens/Home';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

const homeScreenJson = renderer.create(<Home />).toJSON()
const homeScreen = shallow(<Home />)

describe('Home Screen', () => {
  it('renders correctly', () => {
    expect(homeScreenJson).toMatchSnapshot();
  });
})