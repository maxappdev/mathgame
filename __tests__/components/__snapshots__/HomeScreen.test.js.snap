// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Screen renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "height": "100%",
      "justifyContent": "center",
    }
  }
>
  <View
    accessibilityRole="button"
    accessibilityState={Object {}}
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "margin": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        Play
      </Text>
    </View>
  </View>
</View>
`;
